package.json

```diff
@@ -63,11 +63,13 @@
     "@react-aria/i18n": "^3.3.4",
     "@react-aria/interactions": "^3.7.0",
     "@react-aria/listbox": "^3.4.0",
+    "@react-aria/menu": "3.4.3",
     "@react-aria/numberfield": "^3.1.1",
     "@react-aria/overlays": "^3.7.3",
     "@react-aria/radio": "^3.1.6",
     "@react-aria/searchfield": "^3.2.2",
     "@react-aria/select": "^3.6.0",
+    "@react-aria/separator": "3.1.6",
```

Comment:
Looks like we don't use it. Maybe we can add it to the Divider component and add an example with it and Menu to the Storybook.

Reply:
I think it' for separate PR